test/jdk/java/lang/System/FileEncodingTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The year should be `2021, 2022`. (year of inception must be preserved)

Reply:
Oops. Thanks for catching that.